infra/Pos/Util/JsonLog/Events.hs

```diff
@@ -67,15 +65,27 @@ data JLTxS = JLTxS
     , jlsInvReq :: InvReqDataFlowLog
     } deriving Show
 
--- | Get 'SlotId' from 'JLSlotId'.
-fromJLSlotId :: (HasConfiguration, MonadError Text m) => JLSlotId -> m SlotId
-fromJLSlotId (ep, sl) = SlotId (EpochIndex ep) <$> mkLocalSlotIndex sl
+-- | Json log of one transaction being received by a node.
+data JLTxR = JLTxR
+    { jlrTxId  :: Text
+    , jlrError :: Maybe Text
+    } deriving Show
 
--- | Get 'SlotId' from 'JLSlotId'.
-fromJLSlotIdUnsafe :: HasConfiguration => JLSlotId -> SlotId
-fromJLSlotIdUnsafe x = case fromJLSlotId x of
-    Right y -> y
-    Left  _ -> error "illegal slot id"
+-- | Enumeration of all reasons for modifying the mempool.
+data MemPoolModifyReason =
+      -- | Apply a block created by someone else.
+      ApplyBlock
+      -- | Apply a block, with rollback.
+    | ApplyBlockWithRollback
+      -- | Apply a block created by us.
+    -- | CreateBlock
+      -- | Include a transaction. It came from this peer.
+    | ProcessTransaction
+      -- TODO COMMENT
+    -- | Custom Text
+      -- TODO COMMENT
+    -- | Unknown
+    deriving Show
```

Comment:
Let's remove the things that are commented out here.

Reply:
Done.